README
NAME
    Mock::Person::EN - Generate random sets of English names.

SYNOPSIS
     use Mock::Person::EN qw(first_male first_female last_male last_female
             middle_male middle_female name name_female name_male);

     my $first_male = first_male();
     my $first_female = first_female();
     my $last_male = last_male();
     my $last_female = last_female();
     my $middle_male = middle_male();
     my $middle_female = middle_female();
     my $name = name($sex);
     my $female_name = name_female();
     my $male_name = name_male();

DESCRIPTION
    Data for this module was found on these pages:

    Last names
        surnames.behindthename.com
        <http://surnames.behindthename.com/names/usage/english>

    Middle names
        There's usually no distinction between a first and middle name in
        England.

    First names
        Woman: jmenaprijmeni.cz
        <http://www.jmenaprijmeni.cz/anglicka-jmena-zeny> Man:
        jmenaprijmeni.cz <http://www.jmenaprijmeni.cz/anglicka-jmena-muzi>

SUBROUTINES
    "first_male()"
            Returns random first name of male person.

    "first_female()"
            Returns random first name of female person.

    "last_male()"
            Returns random last name of male person.

    "last_female()"
            Returns random last name of female person.

    "middle_male()"
            Returns random middle name of male person.

    "middle_female()"
            Returns random middle name of female person.

    "name([$sex])"
            Recieves scalar with sex of the person ('male' or 'female') and
            returns scalar with generated name. Default value of $sex
            variable is undef, that means random name.

    "name_male()"
            Returns random male name.

    "name_female()"
            Returns random female name.

VARIABLES
    "TYPE"
             Name type.
             Possible values are: 'two', 'three'.
             Default value is 'two'.

EXAMPLE1
     use strict;
     use warnings;

     use Encode qw(encode_utf8);
     use Mock::Person::EN qw(name);

     # Error.
     print encode_utf8(name())."\n";

     # Output like.
     # Mark Parent

EXAMPLE2
     use strict;
     use warnings;

     use Encode qw(encode_utf8);
     use Mock::Person::EN qw(name);

     # Set output name to three names.
     $Mock::Person::EN::TYPE = 'three';

     # Error.
     print encode_utf8(name())."\n";

     # Output like.
     # Jack Ryan Hatheway

EXAMPLE3
     use strict;
     use warnings;

     use Encode qw(encode_utf8);
     use Mock::Person::EN;

     # Get all last male names.
     my @last_males = @Mock::Person::EN::last_male;

     # Print out.
     print sort map { encode_utf8($_)."\n" } @last_males;

     # Output:
     # Abbey
     # Abel
     # Abney
     # Abraham
     # Abrahams
     # Abrahamson
     # Abram
     # Abrams
     # Abramson
     # Achilles
     # Acker
     # Ackerman
     # Adair
     # Adam
     # Adams
     # Adamson
     # Adcock
     # Addison
     # Adkins
     # Aiken
     # Ainsworth
     # Aitken
     # Akerman
     # Akers
     # Albert
     # Alberts
     # Albertson
     # Albinson
     # Alden
     # Alexander
     # Alfredson
     # Alfson
     # Alger
     # Allard
     # Allen
     # Allsopp
     # Alvey
     # Alvin
     # Anderson
     # Andrews
     # Andrewson
     # Ansel
     # Anson
     # Anthonyson
     # Appleby
     # Appleton
     # Archer
     # Arkwright
     # Armistead
     # Arnold
     # Arrington
     # Arterberry
     # Arterbury
     # Arthur
     # Arthurson
     # Ash
     # Ashley
     # Ashworth
     # Atkins
     # Atkinson
     # Attaway
     # Atteberry
     # Atterberry
     # Attwater
     # Audley
     # Augustine
     # Austin
     # Auteberry
     # Autenberry
     # Auttenberg
     # Averill
     # Avery
     # Ayers
     # Aylmer
     # Ayton
     # Babcock
     # Babcocke
     # Babcoke
     # Backus
     # Badcock
     # Badcocke
     # Badcoke
     # Bagley
     # Bailey
     # Baines
     # Baker
     # Baldwin
     # Bancroft
     # Banister
     # Banks
     # Banner
     # Bannerman
     # Barber
     # Bardsley
     # Barker
     # Barlow
     # Barnes
     # Barret
     # Barrett
     # Barton
     # Bartram
     # Bass
     # Bates
     # Bateson
     # Battle
     # Batts
     # Baxter
     # Beake
     # Beasley
     # Beattie
     # Beck
     # Becket
     # Beckett
     # Beckham
     # Belcher
     # Bell
     # Bellamy
     # Benbow
     # Benjaminson
     # Bennet
     # Bennett
     # Benson
     # Bentley
     # Benton
     # Bernard
     # Berry
     # Beverley
     # Beverly
     # Bird
     # Bishop
     # Black
     # Blackbourne
     # Blackburn
     # Blackman
     # Blackwood
     # Blake
     # Blakeslee
     # Bloodworth
     # Bloxam
     # Bloxham
     # Blue
     # Blythe
     # Boivin
     # Bolton
     # Bond
     # Bone
     # Bonham
     # Bonher
     # Bonner
     # Bonney
     # Boon
     # Boone
     # Booner
     # Boothman
     # Botwright
     # Bourke
     # Boyce
     # Braddock
     # Bradford
     # Bradley
     # Brams
     # Bramson
     # Brand
     # Brandon
     # Brant
     # Brasher
     # Brassington
     # Bray
     # Breckenridge
     # Breckinridge
     # Brent
     # Brewer
     # Brewster
     # Brigham
     # Bristol
     # Bristow
     # Britton
     # Broadbent
     # Brock
     # Brook
     # Brooke
     # Brooks
     # Brown
     # Brownlow
     # Bryan
     # Bryant
     # Bryson
     # Buckley
     # Bullard
     # Bulle
     # Bullock
     # Bunker
     # Burke
     # Burnham
     # Burns
     # Burrell
     # Burton
     # Bush
     # Butcher
     # Butler
     # Butts
     # Byrd
     # Cannon
     # Cantrell
     # Carl
     # Carlisle
     # Carlyle
     # Carman
     # Carpenter
     # Carter
     # Cartwright
     # Carver
     # Caulfield
     # Causer
     # Causey
     # Chadwick
     # Chamberlain
     # Chance
     # Chancellor
     # Chandler
     # Chapman
     # Chase
     # Cheshire
     # Christians
     # Christianson
     # Christinsen
     # Christison
     # Christopher
     # Christophers
     # Christopherson
     # Church
     # Clark
     # Clarke
     # Clarkson
     # Clay
     # Clayton
     # Clemens
     # Clifford
     # Clifton
     # Cline
     # Clinton
     # Close
     # Coburn
     # Cock
     # Cockburn
     # Cocks
     # Coel
     # Coke
     # Cokes
     # Colbert
     # Cole
     # Coleman
     # Collingwood
     # Collins
     # Colton
     # Combs
     # Comstock
     # Constable
     # Cook
     # Cooke
     # Cookson
     # Coombs
     # Cooper
     # Corey
     # Cornell
     # Corra
     # Cory
     # Cotterill
     # Courtenay
     # Courtney
     # Cowden
     # Cox
     # Crawford
     # Crewe
     # Croft
     # Cropper
     # Cross
     # Crouch
     # Cummins
     # Curtis
     # Dabney
     # Dalton
     # Dane
     # Danell
     # Daniel
     # Daniell
     # Daniels
     # Danielson
     # Dannel
     # Danniel
     # Danniell
     # Darby
     # Darrell
     # Darwin
     # Daubney
     # Daugherty
     # David
     # Davidson
     # Davies
     # Davis
     # Davison
     # Dawson
     # Day
     # Deadman
     # Dean
     # Dedrick
     # Deering
     # Delaney
     # Denman
     # Dennel
     # Dennell
     # Denzil
     # Derby
     # Derrick
     # Derricks
     # Derrickson
     # Devereux
     # Devin
     # Devine
     # Dexter
     # Dick
     # Dickens
     # Dickenson
     # Dickinson
     # Dickman
     # Dickson
     # Disney
     # Dixon
     # Donalds
     # Donaldson
     # Downer
     # Draper
     # Dudley
     # Duke
     # Dukes
     # Dunn
     # Durand
     # Durant
     # Dustin
     # Dwerryhouse
     # Dwight
     # Dyer
     # Dyson
     # Eads
     # Earl
     # Earls
     # Easom
     # Eason
     # Easton
     # Eaton
     # Eccleston
     # Ecclestone
     # Edgar
     # Edison
     # Edwards
     # Edwardson
     # Elder
     # Eldred
     # Eldridge
     # Eliot
     # Eliott
     # Ellery
     # Elliot
     # Elliott
     # Ellis
     # Ellison
     # Ellisson
     # Elliston
     # Ellsworth
     # Elmer
     # Elwin
     # Elwyn
     # Ely
     # Emerson
     # Emmet
     # Emmett
     # Endicott
     # English
     # Ericson
     # Espenson
     # Ethans
     # Eustis
     # Evanson
     # Evelyn
     # Evered
     # Everett
     # Everill
     # Ewart
     # Fabian
     # Fairbairn
     # Fairburn
     # Fairchild
     # Fairclough
     # Farnham
     # Faulkner
     # Fay
     # Fear
     # Fenn
     # Fields
     # Firmin
     # Fisher
     # Fishman
     # Fitzroy
     # Fleming
     # Fletcher
     # Ford
     # Forest
     # Forester
     # Forney
     # Forrest
     # Foss
     # Foster
     # Fox
     # Frank
     # Franklin
     # Franklyn
     # Freeman
     # Frost
     # Fry
     # Frye
     # Fuller
     # Gabriels
     # Gabrielson
     # Gardenar
     # Gardener
     # Gardiner
     # Gardner
     # Gardyner
     # Garey
     # Garfield
     # Garland
     # Garner
     # Garnet
     # Garnett
     # Garrard
     # Garret
     # Garrett
     # Garrod
     # Garry
     # Gary
     # Geary
     # Georgeson
     # Gibb
     # Gibbs
     # Gibson
     # Giffard
     # Gilbert
     # Giles
     # Gilliam
     # Gladwin
     # Gladwyn
     # Glover
     # Goddard
     # Godfrey
     # Goffe
     # Goode
     # Goodwin
     # Gorbold
     # Gore
     # Granger
     # Grant
     # Granville
     # Gray
     # Green
     # Greene
     # Gregory
     # Grenville
     # Grey
     # Griffin
     # Groves
     # Gully
     # Hackett
     # Hadaway
     # Haden
     # Haggard
     # Haight
     # Hailey
     # Hale
     # Haley
     # Hall
     # Hallman
     # Hambledon
     # Hambleton
     # Hameldon
     # Hamilton
     # Hamm
     # Hampson
     # Hampton
     # Hancock
     # Hanley
     # Hanson
     # Harden
     # Hardwick
     # Hardy
     # Harford
     # Hargrave
     # Harlan
     # Harland
     # Harley
     # Harlow
     # Harman
     # Harmon
     # Haroldson
     # Harper
     # Harrell
     # Harrelson
     # Harris
     # Harrison
     # Hart
     # Hartell
     # Harvey
     # Hathaway
     # Hatheway
     # Hathoway
     # Haward
     # Hawk
     # Hawking
     # Hawkins
     # Hayden
     # Hayes
     # Hayley
     # Hayward
     # Haywood
     # Head
     # Headley
     # Heath
     # Hedley
     # Henderson
     # Hendry
     # Henry
     # Henryson
     # Henson
     # Hepburn
     # Herbert
     # Herberts
     # Herbertson
     # Herman
     # Hermanson
     # Hewitt
     # Hext
     # Hibbert
     # Hicks
     # Hightower
     # Hill
     # Hillam
     # Hilton
     # Hobbes
     # Hobbs
     # Hobson
     # Hodges
     # Hodson
     # Hogarth
     # Hollands
     # Hollins
     # Holme
     # Holmes
     # Holmwood
     # Holt
     # Honeycutt
     # Honeysett
     # Hooker
     # Hooper
     # Hope
     # Hopkins
     # Hopper
     # Hopson
     # Horn
     # Horne
     # Horsfall
     # Horton
     # House
     # Howard
     # Howe
     # Howland
     # Howse
     # Huddleson
     # Huddleston
     # Hudnall
     # Hudson
     # Huff
     # Hughes
     # Hull
     # Hume
     # Hunnisett
     # Hunt
     # Hunter
     # Hurst
     # Hutson
     # Huxley
     # Huxtable
     # Hyland
     # Ianson
     # Ibbot
     # Ibbott
     # Ikin
     # Ilbert
     # Ingham
     # Ingram
     # Irvin
     # Irvine
     # Irving
     # Irwin
     # Isaacson
     # Ivers
     # Jack
     # Jackson
     # Jacobs
     # Jacobson
     # Jakeman
     # James
     # Jamison
     # Jans
     # Janson
     # Jardine
     # Jarrett
     # Jarvis
     # Jeanes
     # Jeffers
     # Jefferson
     # Jeffery
     # Jeffries
     # Jekyll
     # Jenkins
     # Jephson
     # Jepson
     # Jernigan
     # Jerome
     # Jervis
     # Jewel
     # Jewell
     # Jinks
     # Johns
     # Johnson
     # Joiner
     # Jones
     # Jordan
     # Josephs
     # Josephson
     # Joyner
     # Judd
     # Kay
     # Keen
     # Keighley
     # Kellogg
     # Kelsey
     # Kemp
     # Kendal
     # Kendall
     # Kendrick
     # Kennard
     # Kerry
     # Kersey
     # Kevins
     # Kevinson
     # Key
     # Keys
     # Kidd
     # Killam
     # Kimball
     # Kimberley
     # Kimberly
     # King
     # Kingsley
     # Kingston
     # Kipling
     # Kirby
     # Kitchen
     # Kitchens
     # Knaggs
     # Knight
     # Kynaston
     # Lacey
     # Lacy
     # Lamar
     # Landon
     # Lane
     # Langdon
     # Langley
     # Larson
     # Law
     # Lawrence
     # Lawson
     # Layton
     # Leach
     # Leavitt
     # Ledford
     # Lee
     # Leigh
     # Leighton
     # Leon
     # Leonardson
     # Levitt
     # Lewin
     # Lewis
     # Leyton
     # Lincoln
     # Lindon
     # Lindsay
     # Linton
     # Linwood
     # Little
     # Lockwood
     # Loman
     # London
     # Long
     # Longstaff
     # Lovel
     # Lovell
     # Low
     # Lowe
     # Lowell
     # Lowry
     # Lucas
     # Lukeson
     # Lum
     # Lund
     # Lyndon
     # Lynn
     # Lynton
     # Lynwood
     # Lyon
     # Macey
     # Macy
     # Maddison
     # Madison
     # Mallory
     # Mann
     # Marchand
     # Mark
     # Marley
     # Marlow
     # Marsden
     # Marshall
     # Marston
     # Martel
     # Martell
     # Martin
     # Martins
     # Martinson
     # Mason
     # Massey
     # Masters
     # Masterson
     # Mathers
     # Mathews
     # Mathewson
     # Matthews
     # Matthewson
     # May
     # Mayer
     # Mayes
     # Maynard
     # Meadows
     # Mercer
     # Merchant
     # Merrick
     # Merricks
     # Merrickson
     # Merrill
     # Merritt
     # Michael
     # Michaels
     # Michaelson
     # Midgley
     # Milburn
     # Miles
     # Milford
     # Miller
     # Millhouse
     # Mills
     # Milton
     # Mitchell
     # Monday
     # Mondy
     # Montgomery
     # Moore
     # Moores
     # Moors
     # Morce
     # Morison
     # Morris
     # Morrish
     # Morrison
     # Morriss
     # Morse
     # Moses
     # Mottershead
     # Mounce
     # Murgatroyd
     # Muttoone
     # Myers
     # Myles
     # Nathans
     # Nathanson
     # Nelson
     # Ness
     # Neville
     # Newell
     # Newman
     # Newport
     # Newton
     # Nichols
     # Nicholson
     # Nicolson
     # Nielson
     # Nigel
     # Niles
     # Nixon
     # Noel
     # Norman
     # Normanson
     # Norris
     # North
     # Northrop
     # Norwood
     # Nowell
     # Nye
     # Oakley
     # Odell
     # Ogden
     # Olhouser
     # Oliver
     # Oliverson
     # Olson
     # Osborne
     # Osbourne
     # Otis
     # Ott
     # Outlaw
     # Outterridge
     # Overton
     # Owston
     # Paddon
     # Padmore
     # Page
     # Paget
     # Paige
     # Palmer
     # Parent
     # Paris
     # Parish
     # Parker
     # Parris
     # Parsons
     # Paternoster
     # Paterson
     # Patrick
     # Patrickson
     # Patterson
     # Patton
     # Paulson
     # Payne
     # Payton
     # Peacock
     # Peak
     # Pearce
     # Pearson
     # Peck
     # Pelley
     # Pemberton
     # Penny
     # Perkins
     # Perry
     # Peter
     # Peters
     # Peterson
     # Petit
     # Pettigrew
     # Peyton
     # Philips
     # Phillips
     # Pickering
     # Pickle
     # Pierce
     # Pierson
     # Piper
     # Pitts
     # Plank
     # Plaskett
     # Platt
     # Pocock
     # Polley
     # Pond
     # Poole
     # Pope
     # Porcher
     # Porter
     # Potter
     # Pound
     # Power
     # Powers
     # Prescott
     # Pressley
     # Preston
     # Proudfoot
     # Pryor
     # Purcell
     # Putnam
     # Queen
     # Queshire
     # Quick
     # Quickley
     # Quigg
     # Quigley
     # Quincey
     # Quincy
     # Radcliff
     # Radclyffe
     # Raines
     # Rains
     # Rake
     # Rakes
     # Ramsey
     # Randal
     # Randall
     # Randell
     # Ray
     # Rayne
     # Raynerson
     # Read
     # Readdie
     # Ready
     # Reed
     # Reeve
     # Reier
     # Rennell
     # Rennold
     # Rennoll
     # Revie
     # Rey
     # Reynell
     # Reynolds
     # Rhodes
     # Rice
     # Richard
     # Richards
     # Richardson
     # Rickard
     # Rider
     # Ridley
     # Rier
     # Rigby
     # Riley
     # Rimmer
     # Roach
     # Robbins
     # Robert
     # Roberts
     # Robertson
     # Robinson
     # Roderick
     # Rogers
     # Rogerson
     # Rollins
     # Rome
     # Romilly
     # Roscoe
     # Rose
     # Ross
     # Rounds
     # Rowbottom
     # Rowe
     # Rowland
     # Rowntree
     # Roy
     # Royce
     # Royceston
     # Roydon
     # Royle
     # Royston
     # Ruggles
     # Rupertson
     # Rush
     # Ruskin
     # Russel
     # Russell
     # Ryder
     # Rye
     # Ryer
     # Ryers
     # Ryley
     # Sackville
     # Sadler
     # Salomon
     # Salvage
     # Sampson
     # Samson
     # Samuel
     # Samuels
     # Samuelson
     # Sanders
     # Sanderson
     # Sandford
     # Sands
     # Sanford
     # Sangster
     # Sappington
     # Sargent
     # Saunders
     # Sauvage
     # Savage
     # Savege
     # Savidge
     # Sawyer
     # Saylor
     # Scarlett
     # School
     # Scott
     # Scriven
     # Scrivener
     # Scrivenor
     # Scrivens
     # Seabrooke
     # Seaver
     # Selby
     # Sempers
     # Senior
     # Sergeant
     # Sessions
     # Seward
     # Sexton
     # Seymour
     # Shakesheave
     # Sharman
     # Sharrow
     # Shaw
     # Shelby
     # Shepard
     # Sherburne
     # Sherman
     # Shine
     # Sidney
     # Simmons
     # Simms
     # Simon
     # Simons
     # Simonson
     # Simpkin
     # Simpson
     # Sims
     # Sinclair
     # Skinner
     # Slater
     # Smalls
     # Smedley
     # Smith
     # Smythe
     # Snelling
     # Snider
     # Sniders
     # Snyder
     # Snyders
     # Southers
     # Southgate
     # Sowards
     # Spalding
     # Sparks
     # Spear
     # Spearing
     # Spears
     # Speight
     # Spence
     # Spencer
     # Spooner
     # Spurling
     # St John
     # Stacey
     # Stack
     # Stacks
     # Stacy
     # Stafford
     # Stainthorpe
     # Stamp
     # Stanton
     # Stark
     # Starr
     # Statham
     # Steed
     # Steele
     # Steffen
     # Stenet
     # Stephens
     # Stephenson
     # Stern
     # Stevens
     # Stevenson
     # Stidolph
     # Stoddard
     # Stone
     # Strange
     # Street
     # Strickland
     # Stringer
     # Stroud
     # Strudwick
     # Studwick
     # Styles
     # Sudworth
     # Suggitt
     # Summerfield
     # Summers
     # Sumner
     # Sutton
     # Sweet
     # Swindlehurst
     # Sydney
     # Symons
     # Tailor
     # Tanner
     # Tash
     # Tasker
     # Tate
     # Tatham
     # Taylor
     # Teel
     # Tennison
     # Tennyson
     # Terrell
     # Terry
     # Thacker
     # Thatcher
     # Thomas
     # Thompsett
     # Thompson
     # Thomson
     # Thorburn
     # Thorn
     # Thorne
     # Thorpe
     # Thrussell
     # Thurstan
     # Thwaite
     # Tifft
     # Timberlake
     # Timothyson
     # Tinker
     # Tipton
     # Tirrell
     # Tittensor
     # Tobias
     # Tobin
     # Tod
     # Todd
     # Toft
     # Tolbert
     # Tollemache
     # Toller
     # Towner
     # Townsend
     # Tracey
     # Tracy
     # Traiylor
     # Trask
     # Travers
     # Traves
     # Travis
     # Traviss
     # Traylor
     # Treloar
     # Trengove
     # Trent
     # Trevis
     # Triggs
     # Trueman
     # Truman
     # Tucker
     # Tuff
     # Tuft
     # Tupper
     # Turnbull
     # Turner
     # Tyler
     # Tyrell
     # Tyrrell
     # Tyson
     # Underhill
     # Underwood
     # Upton
     # Van Middlesworth
     # Vance
     # Vann
     # Varley
     # Varnham
     # Verity
     # Vernon
     # Victor
     # Victors
     # Victorson
     # Vincent
     # Vipond
     # Virgo
     # Wakefield
     # Walker
     # Wallace
     # Waller
     # Wallis
     # Walmsley
     # Walsh
     # Walter
     # Walterson
     # Walton
     # Ward
     # Wardrobe
     # Ware
     # Warner
     # Warren
     # Warrick
     # Warwick
     # Wash
     # Washington
     # Waterman
     # Waters
     # Watkins
     # Watson
     # Way
     # Wayne
     # Weaver
     # Webb
     # Webster
     # Weekes
     # Wells
     # Wembley
     # Wescott
     # Wesley
     # West
     # Westbrook
     # Westley
     # Wheeler
     # Wheelock
     # Whinery
     # Whitaker
     # White
     # Whitney
     # Whittemore
     # Whittle
     # Wickham
     # Wilbur
     # Wilcox
     # Wilkerson
     # Wilkie
     # Wilkins
     # Wilkinson
     # Willard
     # William
     # Williams
     # Williamson
     # Willis
     # Willoughby
     # Wilmer
     # Wilson
     # Winchester
     # Winfield
     # Winship
     # Winslow
     # Winston
     # Winter
     # Winterbottom
     # Winthrop
     # Winton
     # Witherspoon
     # Wolf
     # Wolfe
     # Womack
     # Wood
     # Woodcock
     # Woodham
     # Woodhams
     # Woods
     # Woodward
     # Wootton
     # Wortham
     # Wragge
     # Wray
     # Wright
     # Wyatt
     # Wyght
     # Wyman
     # Wyndham
     # Wynne
     # Yap
     # Yates
     # Yong
     # York
     # Young
     # Younge
     # Yoxall

DEPENDENCIES
    Exporter, List::Util, Readonly.

SEE ALSO
    Task::Mock::Person
        Install the Mock::Person modules.

REPOSITORY
    <https://github.com/michal-josef-spacek/Mock-Person-EN>

AUTHOR
    Michal Josef Špaček <mailto:skim@cpan.org>

    <http://skim.cz>

LICENSE AND COPYRIGHT
    © Michal Josef Špaček 2013-2023

    BSD 2-Clause License

VERSION
    0.06



Changes
0.06

0.05 2023-11-27T17:34:28+01:00
 - Fix use lib '.' in Makefile.PL.
 - Improve LICENSE AND COPYRIGHT section in doc.
 - Improve SYNOPSIS section in doc.
 - Move bugtracker to github.
 - Rewrite List::MoreUtils to List::Util.
 - Rewrite example files to better names.
 - Update Module::Install to 1.21 version.
 - Update author github username.
 - Update author name.
 - Update copyright years.

0.04 2015-10-06T09:29:19+02:00
 - Fix comment (thanks Mohammad S Anwar).
 - Improve LICENSE AND COPYRIGHT section in doc.
 - Improve SEE ALSO section in doc.
 - Move author tests to xt/ directory.
 - Rewrite B<> to C<> on subroutines and variables.
 - Update Module::Install to 1.16 version.
 - Update copyright years.

0.03 2014-11-09T03:24:25+01:00
 - Fix JOHN to John.
 - Fix NATASHA to Natasha.
 - Fix tests for 3/4 word names.
 - Update Module::Install to 1.14 version.

0.02 2014-07-24T08:14:53+02:00
 - Fix list of last names.

0.01 2014-07-21T14:53:00+02:00
 - First version.


Makefile.PL
use lib '.';
use strict;
use warnings;

use inc::Module::Install;

# Definition.
abstract 'Generate random sets of English names.';
author 'Michal Josef Spacek <skim@cpan.org>';
author_requires 'English' => 0;
author_requires 'File::Object' => 0.05;
author_requires 'Test::More' => 0;
author_requires 'Test::NoWarnings' => 0;
author_requires 'Test::Pod' => 0;
author_requires 'Test::Pod::Coverage' => 0;
license 'bsd';
name 'Mock-Person-EN';
readme_from 'EN.pm';
recursive_author_tests('xt');
requires 'Exporter' => 0;
requires 'List::Util' => 1.33;
requires 'Readonly' => 0;
requires 'perl' => '5.8.0';
resources 'bugtracker' => 'https://github.com/michal-josef-spacek/Mock-Person-EN/issues';
resources 'homepage' => 'https://github.com/michal-josef-spacek/Mock-Person-EN';
resources 'repository' => 'git://github.com/michal-josef-spacek/Mock-Person-EN';
test_requires 'List::Util' => 1.33;
test_requires 'Test::More' => 0;
test_requires 'Test::NoWarnings' => 0;
tests_recursive;
version '0.06';

# Run.
WriteAll();
